lib/logstash/inputs/http_poller.rb

```diff
@@ -233,27 +267,37 @@ def handle_failure(queue, name, request, exception, execution_time)
   private
   def apply_metadata(event, name, request, response=nil, execution_time=nil)
```

Comment:
minor: we always pass in all the arguments ...
```suggestion
  def apply_metadata(event, name, request, response, execution_time)
 ```